docs/source/examples/mg-mode-part-i.ipynb

```diff
@@ -0,0 +1,926 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fd19aa6f780>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "root_universe.plot(center=(0., 0., 0.), width=(3 * 21.42, 3 * 21.42), pixels=(500, 500),\n",
+    "                   color_by='material')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "OK, it looks pretty good, lets go ahead and write the file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create Geometry and set root Universe\n",
+    "geometry = openmc.Geometry()\n",
+    "geometry.root_universe = root_universe\n",
+    "\n",
+    "# Export to \"geometry.xml\"\n",
+    "geometry.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now create the tally file information. The tallies will be set up to  give us the pin powers in this notebook. We will do this with a mesh filter, with one mesh cell per pin."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "tallies_file = openmc.Tallies()\n",
+    "\n",
+    "# Instantiate a tally Mesh\n",
+    "mesh = openmc.Mesh()\n",
+    "mesh.type = 'regular'\n",
+    "mesh.dimension = [17 * 2, 17 * 2]\n",
+    "mesh.lower_left = [-32.13, -10.71]\n",
+    "mesh.upper_right = [+10.71, +32.13]\n",
+    "\n",
+    "# Instantiate tally Filter\n",
+    "mesh_filter = openmc.MeshFilter(mesh)\n",
+    "\n",
+    "# Instantiate the Tally\n",
+    "tally = openmc.Tally(name='mesh tally')\n",
+    "tally.filters = [mesh_filter]\n",
+    "tally.scores = ['fission']\n",
+    "\n",
+    "# Add tally to collection\n",
+    "tallies_file.append(tally)\n",
+    "\n",
+    "# Export all tallies to a \"tallies.xml\" file\n",
+    "tallies_file.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the geometry and materials finished, we now just need to define simulation parameters for the `settings.xml` file. Note the use of the `energy_mode` attribute of our `settings_file` object. This is used to tell OpenMC that we intend to run in multi-group mode instead of the default continuous-energy mode. If we didn't specify this but our cross sections file was not a continuous-energy data set, then OpenMC would complain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 1000\n",
+    "inactive = 100\n",
+    "particles = 5000\n",
```

Comment:
I still hesitate to go too far with this; I will bring it down to the levels @smharper suggests, but I dont want to go much further.  These notebooks are about the users not the developers. If we want these to be tests for developers, they should be tests, but the will be different code in the end.